### PR TITLE
qa: Initial kraken-x checkin

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2017-02-09_22:09:46-upgrade:kraken-x-master-distro-basic-vps/

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com/yuriw-2017-02-10_16:46:42-upgrade:kraken-x-luminous-distro-basic-smithi/

File: qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-all.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+   sequential:
+   - ceph.restart:
+       daemons: [mon.a, mon.b, mon.c, mds.a, osd.0, osd.1, osd.2, osd.3]
+       wait-for-healthy: false
+       wait-for-osds-up: true
+   - exec:
+       mon.a:
+         - ceph osd set require_luminous_osds
+   - ceph.restart:
+       daemons: [osd.0]
```
Comment (jdurgin) [inline qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-all.yaml]: already restarted above

File: qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+meta:
+- desc: |
+   upgrade the ceph cluster,
+   upgrate in two steps 
+   step one ordering: mon.a, osd.0, osd.1, mds.a
```
Comment (jdurgin) [inline qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml]: can get rid of this whole upgrade sequence - mons have to be first

File: qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-all.yaml (hunk 1 above)
Comment (yuriw) [inline qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-all.yaml]: done

File: qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml (hunk 2 above)
Comment (yuriw) [inline qa/suites/upgrade/kraken-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml]: removed upgrade-mon-osd-mds.yaml

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2017-02-13_20:46:52-upgrade:kraken-x-luminous-distro-basic-smithi/

**Comment by liewegas** [comment]
updated/fixed here: https://github.com/ceph/ceph/pull/13517

**Pull request closed without merging**